pkg/server/log.go

```diff
@@ -0,0 +1,29 @@
+package server
+
+import (
+	"io"
+	"log"
+	"strings"
+)
+
+type FilteredWriter struct {
+	io.Writer
+}
+
+var msgsToFilter = []string{
```

Comment:
If teresa-server is flooded with a whole lot of HTTP requests without TLS, this change would hide that from us.

We should try to fix the ELB behavior instead.